Add Active Directory infrastructure support

f-trivino opened the pull request:
This commit implements new "provision_ad.yml" ansible playbook and new roles
such as packages, win-common, win-dns, win-domain, win-forest, win-ping, and
win-shcema. A new class "RunADTests" has been implemented to execute AD test
suites, and a new vagrant file "Vagrantfile.ad" defines the AD cluster, which
is composed by controller, ad, ad-child, master, and client0 VMs.

The implementation is based on three steps: a) prci deploy changes (new
packages and dependencies requirements, such as winrm modules), b) vagrant
cluster (using well-defined images for both Linux and Windows machines), and
c) software provisioning (software, schemas, the configuration of the scenario)

netoarmando commented:
I've removed the unnecessary role named 'packages', updated the commit message to reflect that and added some CRLF add the end of files.

f-trivino commented:
> I've removed the unnecessary role named 'packages', updated the commit message to reflect that and added some CRLF add the end of files.

great!!
